NEWS.md

```diff
@@ -12,7 +12,13 @@
 
 ### Changed
```

Comment:
Please add your changes under the Developing section, not the old release.

Reply:
Oh, I missed that. Fixed now.